lib/linguist/languages.yml

```diff
@@ -303,6 +305,7 @@ Bison:
   extensions:
   - .bison
   ace_mode: text
+  color: "#6A463F"
```

Comment:
This language is in the group Yacc so it will be displayed as Yacc in the statistics bar. I don't think it's necessary to add a language for it ;)